test/source/mock/attester/attester-endpoints.ts

```diff
@@ -98,6 +104,24 @@ export const mockAttesterEndpoints: HandlersDefinition = {
       throw new HttpClientErr(`Not implemented: ${req.method}`);
     }
   },
+  '/attester/ldap-relay': async (parsedReq, req) => {
+    const server = parsedReq.query.server;
+    const emailOrLongid = parsedReq.query.search;
+    if (isGet(req)) {
+      if (emailOrLongid === 'test.ldap.priority@gmail.com') {
+        return protonMailCompatKey;
+      }
+      if (emailOrLongid === 'test.flowcrypt.pubkeyserver.priority@gmail.com') {
+        return protonMailCompatKey;
+      }
+      if (emailOrLongid === 'test.ldap.keyserver.pgp@gmail.com' && server === 'keyserver.pgp.com') {
+        return [protonMailCompatKey, testMatchPubKey].join('\n');
+      }
+      throw new HttpClientErr('No OpenPGP LDAP server on this address.', 404);
```

Comment:
Please use constant `Status.NOT_FOUND` instead of 404.

I suggest to also add cases that emulate request timeout
( with `throw new HttpClientErr('RequestTimeout', Status.BAD_REQUEST);`)
before and after the successful fetch.

Reply:
That is,
1) the priority (customer-specific LDAP) source timed out -- others returned something.
2) the priority source returned something, another source timed out.

Reply:
Yeah. that test is must to have.
Just added test.